docs/headless.md

```diff
@@ -57,10 +57,11 @@ Building your buyer portal application requires you to run the `yarn build` comm
 Make sure that you have configured the following `.env` values correctly before building:
 
 - `VITE_IS_LOCAL_ENVIRONMENT`: Set this to `FALSE`
-- `VITE_ASSETS_ABSOLUTE_PATH`: Set this to the URL where the assets folder is hosted. **Note that this needs to be the absolute URL to the `/assets` folder location where the build will be served from in production.** Also please include the trailing `/`.
-- `VITE_DISABLE_BUILD_HASH`: Set this to `TRUE` if you want to disable the hash in the build files. This is useful if you want to avoid updating the file names in your headless app everytime you deploy or if you are using a CDN that does not support cache busting with hashes.
+- `VITE_ASSETS_ABSOLUTE_PATH`: Set this to the URL where the assets folder is hosted. Also please include the trailing `/`.
+
+  For example, if you deploy the contents of the `dist` folder built by running `yarn build` and hosted it at https://my.custom.cdn/generated/b2b, the value you should put is https://my.custom.cdn/generated/b2b/.
 
-  For example, if you deploy the contents of the `dist` folder built by running `yarn build` and hosted it at https://my.custom.cdn/generated/b2b, the value you should put is https://my.custom.cdn/generated/b2b/assets/.
+- `VITE_DISABLE_BUILD_HASH`: Set this to `TRUE` if you want to disable the hash in the build files. This is useful if you want to avoid updating the file names in your headless app everytime you deploy or if you are using a CDN that does not support cache busting with hashes.
```

Comment:
I thought that hashes help with cache busting - but without hashes, you do need a CDN that supports cache busting. 

The no hashes option is more useful for people deploying buyer portal separately from their storefront, so they can just include the same file consistently. I.e `https://my.custom.cdn/generated/b2b/index.js` instead of having to keep changing the included filenames to `https://my.custom.cdn/generated/b2b/index.abcdef.js`. I know we use the entry point system to have.

```suggestion
- `VITE_DISABLE_BUILD_HASH`: Set this to `TRUE` if you want to disable the hash in the build files. This is useful if you want to avoid updating the file names in your headless app on each deployment and you would like to have a stable reference.
```